libs/payments/eligibility/project.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "payments-eligibility",
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "libs/payments/eligibility/src",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@nx/js:tsc",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "rootDir": ".",
+        "outputPath": "dist/libs/payments/eligibility",
+        "tsConfig": "libs/payments/eligibility/tsconfig.lib.json",
+        "packageJson": "libs/payments/eligibility/package.json",
+        "main": "libs/payments/eligibility/src/index.ts",
+        "assets": ["libs/payments/eligibility/*.md"]
+      }
+    },
+    "lint": {
+      "executor": "@nx/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": [
+          "libs/payments/eligibility/**/*.ts",
+          "libs/payments/eligibility/package.json"
+        ]
+      }
+    },
+    "test-unit": {
```

Comment:
When I ran the test command per the generated README, `nx test-unit payments-eligibility`, I got a code coverage warning from the generated `jest.config.ts` file[1] -- Is this expected? Are we doing something with code coverage thus far?

[1]:
```
Validation Warning:

  Unknown option "coverageDirectory" with value "../../../coverage/libs/payments/eligibility" was found.
  This is probably a typing mistake. Fixing it will remove this message.

  Configuration Documentation:
  https://jestjs.io/docs/configuration
```

Reply:
/cc @Trinaa @ashrivastava-qa re: exciting code coverage convo

Reply:
This is expected at the moment. The error can safely be ignored.